lib/aurora_bootstrapper/notifier.rb

```diff
@@ -0,0 +1,55 @@
+module AuroraBootstrapper
+  class Notifier
+    def initialize( s3_path: s3_path)
+        @s3_path = s3_path
+        if ENV.fetch( 'EXPORT_DATE_OVERRIDE', false )
+          @export_date ||= ENV.fetch( 'EXPORT_DATE', DateTime.now.strftime("%Y-%m-%d") )
```

Comment:
does this mean that if we have `EXPORT_DATE_OVERRIDE` unset, we will not set `@export_date` regardless of the value of `EXPORT_DATE`?

Reply:
yup

Reply:
It will affect BI team

Reply:
what could be the case when EXPORT_DATE_OVERRIDE is unset and EXPORT_DATE is set?

Reply:
We should remove EXPORT_DATE altogether and only use EXPORT_DATE_OVERRIDE and current day. The logic will be simpler to follow. There is no reason to put data files under datetime folder other than NOW

Reply:
So just to make sure I understand
The options are:
```sh
# EXPORT_DATE_OVERRIDE unset
# EXPORT_DATE unset
```
no export date will be set.

```sh
export EXPORT_DATE_OVERRIDE=true
export EXPORT_DATE=date
```
export date set to the var

```bash
export EXPORT_DATE_OVERRIDE=true
# EXPORT_DATE unset
```
will auto set the date

```bash
# EXPORT_DATE_OVERRIDE unset
export EXPORT_DATE=date
```
__no export date will be set__
Is this expected?

Reply:
Theoretically this should never happen; we could throw exception

Reply:
If OVERRIDE flag is set and some user specifies EXPORT_DATE, he/she should have get Error

Reply:
OVERRIDE flag shows intention of using datetime folder

Reply:
Or you want to be backward compatible??

Reply:
basck compat is ideal

Reply:
I already updated as you said. Count you check line 8?

Reply:
would this be easier to follow if it were
```ruby
def export_date
  @export_date ||= ENV.fetch('EXPORT_DATE', export_date_override )
end

def export_date_override
  DateTime.now.strftime("%Y-%m-%d") if ENV.[ 'EXPORT_DATE_OVERRIDE' ]
end
```

Reply:
Done